AUTHORS

```diff
@@ -56,6 +56,7 @@ Travis Sikes (@tsikes)
 Harsh Sinha (@sin-ha)
 David Sondak
 Raymond Speth (@speth), Massachusetts Institute of Technology
+Su Sun (@ssun30), Northeastern University
```

Comment:
What order are you putting this in?

Reply:
Per top of file:
> Below is a partial, alphabetical list of developers and contributors to Cantera over the years.

Reply:
But my name is immediately below this? I'm all for alphabetizing but it should be done throughout IMO

Reply:
Hmm - your name is between Walker and Wehrfritz in the document I am looking at? My reading is that the list is alphabetical by last name?

Reply:
Oh, darn it 😔 you're right, thank you!